app/views/smart_answers/_result.html.erb

```diff
@@ -23,15 +23,11 @@
 
 <script type="text/javascript">
   var flowName = '<%= @name %>';
-  event = {
-    hitType: 'event',
-    eventCategory: 'Smart Answer',
-    eventAction: 'Completed',
-    eventLabel: flowName,
-    nonInteraction: 1,
-    page: document.location.pathname
-  };
-  if (typeof window.ga === "function") {
-    window.ga('send', event);
+  var page = document.location.pathname;
```

Comment:
Not sure if it's worth worrying about, but this makes `page` and `options` variables available globally. They're rather generic names and if I'm correct it could in theory cause problems with other JS that may rely on `if(options === undefined)` or similar code. Putting it in a closure would avoid this problem e.g.:
```JavaScript
(function(){
  var flowName = '<%= @name %>';
  var page = document.location.pathname;
  var options = {
    label: flowName,
    nonInteraction: true,
    page: page
  };
  GOVUK.analytics.trackEvent('Smart Answer', 'Completed', options);
})();
```